src/provider/cloud-element-templates/cmd/ChangeElementTemplateHandler.js

```diff
@@ -579,7 +580,7 @@ export default class ChangeElementTemplateHandler {
       }
 
       // (3) add new zeebe:Property
-      else if (shouldUpdate(newPropertyValue, newProperty)) {
+      else if (newPropertyValue && shouldUpdate(newPropertyValue, newProperty)) {
```

Comment:
Should we test for existence of `newPropertyValue` here? It could be an empty string, too.